operators/test/e2e/apm/standalone_test.go

```diff
@@ -28,3 +29,26 @@ func TestApmStandalone(t *testing.T) {
 	test.Sequence(nil, test.EmptySteps, apmBuilder).
 		RunSequential(t)
 }
+
+func TestApmStandaloneNoTLS(t *testing.T) {
+	apmBuilder := apmserver.NewBuilder("standalone-no-tls").
+		WithNamespace(test.Namespace).
+		WithVersion(test.ElasticStackVersion).
+		WithRestrictedSecurityContext().
+		WithOutput(apmtype.Output{}).
+		WithConfig(map[string]interface{}{
+			"output.console": map[string]interface{}{
+				"pretty": true,
+			},
+		}).
+		WithHTTPCfg(v1alpha1.HTTPConfig{
+			TLS: v1alpha1.TLSOptions{
+				SelfSignedCertificate: &v1alpha1.SelfSignedCertificate{
+					Disabled: true,
+				},
+			},
+		})
+
+	test.Sequence(nil, test.EmptySteps, apmBuilder).
```

Comment:
`test.Sequence(nil, test.EmptySteps, builder)` < I'm the author of this but I find this ugly now that I read it again. If you ever have an idea to improve it, it would be welcome.

Reply:
Yes I was thinking about this as well. The difference between `nil` and `EmptySteps` is odd. Maybe it could be a map with well known step names?
```golang
seq := test.Sequence(builder1, builder.2)
seq["pre-init"] = myCustomInitStep()
seq["when-running"] = myCustomFailureTest()
```

Reply:
Let's take care of that in another PR. 
I have a slight preference for fluent APIs.
```go
test.
  Pre(preSteps).
  Steps(steps).
  With(builders).
  RunSequential()
```

Reply:
That's nice too. 👍 on a separate PR